include/c_common/arrays_input.h

```diff
@@ -31,10 +31,20 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #include <postgres.h>
 #include <utils/array.h>
 
-/** @brief enforces the input array to be @b NOT empty */
+/** 
+ * @brief Enforces the input array to be @b NOT empty 
+ * @param *arrlen Length of the array
```

Comment:
Put the everything that is not the `@brief` on the c file
The header should only contain the `@brief`